### ISSUE TITLE
LOG4j Security issues:

### ISSUE THREAD

**Issue by HowardLander** [EBISPOT/ontotools-docker#11]
I noticed that the maintainers of EBISPOT/OLS have asserted that they don't believe that there are any LOG4J worries for that product [link](https://github.com/EBISPOT/OLS/issues/527).

Do we know if that is true for this repo as well?

Thanks
Howard

**Comment by udp** [comment]
Yes, I believe so. In addition to OLS, this repo includes OXO and ZOOMA:

* OXO uses spring boot slf4j which is [not affected](https://spring.io/blog/2021/12/10/log4j2-vulnerability-and-spring-boot).
* ZOOMA uses log4j 1.x which is also not affected.

**Comment by HowardLander** [comment]
Thanks so much for the reply. I'm tempted to leave this open for others to find, but feel free to close it if you think that's better.

Howard
